libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.java

```diff
@@ -177,25 +179,34 @@ public boolean isValidRouteProfile(String routeProfile) {
    */
   @Nullable
   public BannerInstructions findCurrentBannerInstructions(LegStep currentStep, double stepDistanceRemaining) {
-    if (isValidStep(currentStep) && hasInstructions(currentStep.bannerInstructions())) {
-      int roundedDistanceRemaining = (int) stepDistanceRemaining;
-      List<BannerInstructions> instructions = new ArrayList<>(currentStep.bannerInstructions());
-      Iterator<BannerInstructions> instructionsIterator = instructions.iterator();
-      while (instructionsIterator.hasNext()) {
-        BannerInstructions instruction = instructionsIterator.next();
+    if (isValidBannerInstructions(currentStep)) {
+      List<BannerInstructions> instructions = getBannerInstructions(currentStep);
```

Comment:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs. What about `sortBannerInstructions` as you suggested in the first place?
Also, we could change the variable name (`instructions`) to `sortedInstructions`. IMO it reflects better that those are the sorted ones and therefore the code is easier to read and understand. What do you think?